tests/nrf5340_audio/contin_array/Kconfig

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+module = CONTIN_ARRAY
```

Comment:
How are these Kconfig files used/included in the tests?

Reply:
same as in applications/libraries, there's no difference.

Reply:
I dont se any LOG_(...) in the test .c files, but maybe zassert_equal() etc uses these log levels?

Reply:
you're compiling application sources in the test (see CMakeLists.txt), any Kconfig dependencies on that source file need to be part of the test as well. We should avoid doing this and use re-usable libraries instead, but that can be improved later.